<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequirePswForm</class>
 <widget class="QWidget" name="RequirePswForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>247</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RequirePswForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>36</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:Tahoma;
font-style: normal;
font-size: 12pt;
font-weight: bold;

font-size:14px;
color:blue;</string>
   </property>
   <property name="text">
    <string>Введите пароль для секретного ключа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>121</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPswd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;
background-image: url(&quot;Image/ok.png&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;
background-image: url(&quot;Image/cancel.png&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
   </property>
   <property name="text">
    <string>    Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>3</y>
     <width>76</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;Image/passw.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
